.circleci/config.yml

```diff
@@ -27,8 +49,47 @@ jobs:
             . .venv/bin/activate
             mkdir test-results
             pytest --junitxml=test-results/junit.xml
+      - run:
+          name: build_wheel
+          command: |
+            . .venv/bin/activate
+            python3 setup.py sdist bdist_wheel
       - store_test_results:
           path: test-results
 
+      - persist_to_workspace:
+          root: .
+          paths:
+            dist/*
       - store_artifacts:
           path: test-results
+
+  publish:
+    docker:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+    steps:
+      - checkout
+      - *restore_venv
+      - run:
+          name: verify tag
+          command: |-
```

Comment:
are you sure about `|-`?
from docs:
```
- run:
    command: |
      echo Running test
      mkdir -p /tmp/test-results
      make test
```

Reply:
I thought `|` didn't chomp leading space and `|-` did, but yes either way this works - https://app.circleci.com/jobs/github/astronomer/astronomer-airflow-version-check/49

Reply:
Ah https://yaml-multiline.info/ `-` is the Block Chomping Indicator` -- so makes no really difference in  this case as empty lines aren't significant to bash.